executor/pkg_test.go

```diff
@@ -41,6 +41,23 @@ func (m *MockExec) Next(ctx context.Context) (Row, error) {
 	return r, nil
 }
 
+func (m *MockExec) NextChunk(ctx context.Context, chk *chunk.Chunk) error {
+	chk.Reset()
+	rt := m.retTypes()
+	for m.curRowIdx < len(m.Rows) {
+		curRow := m.Rows[m.curRowIdx]
+		for i := 0; i < len(curRow); i++ {
+			curDatum := curRow.GetDatum(i, rt[i])
+			chk.AppendDatum(i, &curDatum)
```

Comment:
```go
chk.AppendDatum(i, &m.Rows[m.curRowIdx].GetDatum(i, rt[i]))
```

Reply:
cannot take the address of `m.Rows[m.curRowIdx].GetDatum(i, rt[i])`.

https://stackoverflow.com/questions/10535743/address-of-a-temporary-in-go
https://golang.org/ref/spec#Address_operators